pysac/mhs_atmosphere/parameters/model_pars.py

```diff
@@ -22,16 +22,16 @@
 hmi_model['chratio'] = 1*u.one - hmi_model['coratio'] - hmi_model['phratio']
 
 mfe_setup = {'photo_scale': 0.60*u.Mm,
-             'chrom_scale': 0.385*u.Mm,
-             'corona_scale': 2*u.Mm,  #scale height for the corona
-             'coratio': 0.02*u.one,
+             'chrom_scale': 0.4*u.Mm,
+             'corona_scale': 0.25*u.Mm,  #scale height for the corona
+             'coratio': 0.0*u.one,
              'model': 'mfe_setup',
              'phratio': 0.0*u.one,
              'pixel': 0.36562475*u.Mm,  #(HMI pixel)
-             'radial_scale': 0.044*u.Mm,
+             'radial_scale': 0.14*u.Mm,
              'nftubes': 1,
-             'B_corona': 4.35e-4*u.T,
-             'pBplus': 4.250e-4*u.T}
+             'B_corona': 4.85e-4*u.T,
+             'pBplus': 0.0e-4*u.T}
```

Comment:
In fact for absolute clarity mfe_setup now has

B_0z = exp(-z/0.4Mm)
B_z = 128.5mT * B_0z^2 * G + 0.485mT

Fred
 
On 08/07/2015 04:07 PM, Frederick Gent wrote:
> No the background field is
> 'B_corona'
>
> It removes the enhancement to the pressure, which was introduced for
> the corona in Paper41, but was never applied by Viktor
> 'pBplus'
>
> On 08/07/2015 04:05 PM, Stuart Mumford wrote:
>>
>> In pysac/mhs_atmosphere/parameters/model_pars.py
>> <https://github.com/SWAT-Sheffield/pysac/pull/9#discussion_r36516308>:
>>
>> >               'nftubes': 1,
>> > -             'B_corona': 4.35e-4*u.T,
>> > -             'pBplus': 4.250e-4*u.T}
>> > +             'B_corona': 4.85e-4*u.T,
>> > +             'pBplus': 0.0e-4*u.T}
>>
>> So this removes the background field??
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/SWAT-Sheffield/pysac/pull/9/files#r36516308>.
>>
>